sdk/python/sawtooth_processor_test/message_factory.py

```diff
@@ -176,7 +177,7 @@ def create_tp_process_request(self, payload, inputs, outputs, deps,
             payload=payload,
             signature=signature)
 
-    def create_batch(self, transactions):
+    def create_batch_object(self, transactions):
```

Comment:
Why the name change?

Reply:
I had thought that I needed to get the batch id to get a status, but I am getting the '--wait' from the identity cli change. The change you called out is unnecessary.